bundle/compliance/cis_azure/rules/cis_7_1/rule.rego

```diff
@@ -0,0 +1,22 @@
+package compliance.cis_azure.rules.cis_7_1
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+
+finding = result {
+	# filter
+	data_adapter.is_bastion
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(evaluation),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+default evaluation := false
+
+evaluation {
+	some i
+	data_adapter.bastions[i].id != ""
```

Comment:
are we sure it will be an empty string?

Reply:
I am just checking the id exists and is not empty as a sanity check, it's the one field that should always be there.